<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BaseAddNetworkLayerWidget</class>
 <widget class="QWidget" name="BaseAddNetworkLayerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>791</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QGridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="_name"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Server:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="_server"/>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="capabilitiesButton">
       <property name="text">
        <string>Get Layers</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="getServiceNamesButton">
       <property name="text">
        <string>Get Service Names</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="_serviceNameLabel">
       <property name="text">
        <string>Service Name:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="_serviceNameLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="_layersGroupBox">
     <property name="title">
      <string>Layers</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QTreeWidget" name="_layersTree">
        <column>
         <property name="text">
          <string>1</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QCheckBox" name="_addAllAsGroup">
       <property name="text">
        <string>Add All As Group</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="_addSelectedAsGroup">
       <property name="text">
        <string>Add Selected As Group</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Image Type</string>
     </property>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QRadioButton" name="_pngButton">
        <property name="text">
         <string>image/png</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="_tifButton">
        <property name="text">
         <string>image/tif</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="_jpegButton">
        <property name="text">
         <string>image/jpeg</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="viewOptionsButton">
       <property name="text">
        <string>View Options</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Cache Directory:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="_cacheDirectory"/>
     </item>
     <item>
      <widget class="QPushButton" name="browseDirectory">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QCheckBox" name="_makeDefaultDirectory">
       <property name="text">
        <string>Make default cache directory</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
